brayns/Brayns.cpp

```diff
@@ -283,14 +336,30 @@ struct Brayns::Impl
     KeyboardHandler& getKeyboardHandler() { return *_keyboardHandler; }
     AbstractManipulator& getCameraManipulator() { return *_cameraManipulator; }
 private:
-    void _render(const Vector2ui& windowSize)
+    bool _render(const Vector2ui& windowSize)
     {
         _engine->reshape(windowSize);
         _engine->preRender();
 
 #if (BRAYNS_USE_DEFLECT || BRAYNS_USE_NETWORKING)
         _executePlugins(windowSize);
 #endif
+
+        if (_dataLoadingFuture.valid())
+        {
+#ifdef BRAYNS_USE_LUNCHBOX
+            if (!_parametersManager->getApplicationParameters()
+                     .getSynchronousMode() &&
```

Comment:
it's the second occurrence of this call which does not fit on a single line, a helper function would be nicer (_useSynchronousMode())